phpseclib/Net/SFTP.php

```diff
@@ -3025,8 +2857,8 @@ private function parseTime($key, $flags, &$response)
     {
         $attr = [];
         list($attr[$key]) = Strings::unpackSSH2('Q', $response);
-        if ($flags & NET_SFTP_ATTR_SUBSECOND_TIMES) {
-            list($attr[key . '-nseconds']) = Strings::unpackSSH2('N', $response);
```

Comment:
Check out this typo! `key` is suppose to be `$key`

Reply:
Nice find!